Makefile

```diff
@@ -79,8 +79,6 @@ check_tools:
 get_tools:
 	@echo "--> Installing tools"
 	./scripts/get_tools.sh
-	@echo "--> Downloading linters (this may take awhile)"
-	$(GOPATH)/src/github.com/alecthomas/gometalinter/scripts/install.sh -b $(GOBIN)
```

Comment:
If you add back these two lines, this should resolve https://github.com/tendermint/tendermint/pull/2824#issuecomment-438166072 AFAIK.

Reply:
this line fails when installing on RPi: https://github.com/alecthomas/gometalinter/blob/master/scripts/install.sh#L62

should we add a platform check for this line?

Reply:
> should we add a platform check for this line?

yes (with a comment saying it does not work on ...)

Reply:
the better option (seemingly) was to split the makefile targets - this is how it is in the SDK now